sirepo/auth/__init__.py

```diff
@@ -411,31 +396,19 @@ def reset_state():
     _set_log_user()
 
 
-def role_for_sim_type(sim_type):
-    return 'sim_type_' + sim_type
-
-
-def set_user_for_utils(uid=None):
-    """A mock user for utilities"""
-    cookie.set_cookie_for_utils()
-    import sirepo.auth.guest
-    if uid:
-        _login_user(sirepo.auth.guest, uid)
-    else:
-        login(sirepo.auth.guest, is_mock=True)
-
 
 @contextlib.contextmanager
-def set_user(uid):
-    """Set the user (uid) for the context"""
-
-    assert not util.flask_app(), \
-        'Flask sets the user on the request'
-    try:
-        set_user_for_utils(uid=uid)
+def set_user_outside_of_flask_request(uid=None):
```

Comment:
Let's call it `set_user_outside_of_http_request`. This will be true in tornado, too, because it's really the concept of cookies that is important here.